app/views/shopping_lists/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class="container">
+  <h1 class="text-center display-4">Shopping List</h1>
+  <div class="d-flex justify-content-between">
+    <div class="h2">Amount of food items to buy: <span class="h3"><%= @foods.length %></span></div>
+    <div class="h2">Total value of food needed: <span class="h3">$ <%= @total_price * -1 %></span></div>
+  </div>
+
+  <div>
+    <% if @foods.length.zero? %>
+      <h2 class="text-center h4 bg-success text-white">All ingredients are set to make the recipe!</h2>
+    <% else %>
+      <table class="table table-bordered table-striped">
+        <thead>
+          <tr class="h2">
+            <th>Food</th>
+            <th>Quantity</th>
+            <th>Total price</th>
+          </tr>
+        </thead>
+        <tbody>
+          <% @foods.each do |food| %>
+            <tr>
+              <td><%= food[:name] %></td>
+              <td><%= food[:quantity] * -1 %></td>
+              <td><%= food[:price] * -1 %></td>
+            </tr>
+          <% end %>
+        </tbody>
+      </table>
+    <% end %>
+  </div>
```

Comment:
- [ ] according to project requirement, when I add food to a shopping list and click generate shopping list I should have a page that has all the food I added, currently your page displays none
your page :point_down: 
![Screenshot from 2023-09-22 21-22-33](https://github.com/sahlminkok/recipe-app/assets/66097371/17b97e57-ada1-472a-8470-8e24e382414f)
design guide :point_down: 
![Screenshot from 2023-09-22 21-21-53](https://github.com/sahlminkok/recipe-app/assets/66097371/7603857a-9b94-4d63-9e6c-92269c7fc8de)

Reply:
thanks alot it fixed 

![image](https://github.com/sahlminkok/recipe-app/assets/110924992/93fadf6d-1188-469d-a6ca-5d07448a4f45)